whylog/config/filename_matchers.py

```diff
@@ -25,6 +30,19 @@ def get_matched_files(self):
             # TODO: finding files in others hosts
             raise NotImplementedError
 
+    def is_belong_to_matcher(self, line_source):
```

Comment:
I suggest to change name into `does_belong_to_matcher` (I think it would be more in English)